test/metabase/lib/test_util.cljc

```diff
@@ -0,0 +1,64 @@
+(ns metabase.lib.test-util
```

Comment:
Added new test util namespace for stuff I found myself using in a bunch of different tests.